pp/integrations/plugin-packs/procedures/hardware-storage-datacore-api.md

```diff
@@ -0,0 +1,404 @@
+| APIUSERNAME     | Username to access the endpoint.                                                                                          |                   | X           |
+| APIPASSWORD     | Password to access the endpoint.                                                                                          |                   | X           |
+| APIPROTO        | HTTP protocol, either http or https                                                                                       | https             |             |
+| APIPORT         | Port of the resource to connect to                                                                                        | 443               |             |
+| APIEXTRAOPTIONS | Any extra option you may want to add to every command (a --verbose flag for example). All options are listed [here](#available-options). |                   |             |
+
+5. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The host appears in the list of hosts, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the host: it shows the values of the macros.
+
+### Using a service template provided by the connector
+
+1. If you have used a host template and checked **Create Services linked to the Template too**, the services linked to the template have been created automatically, using the corresponding service templates. Otherwise, [create manually the services you want](/docs/monitoring/basic-objects/services) and apply a service template to them.
+2. Fill in the macros you want (e.g. to change the thresholds for the alerts). Some macros are mandatory (see the table below).
+
+<Tabs groupId="sync">
+<TabItem value="Alerts" label="Alerts">
+
+| Macro           | Description                                                                                                                                                    | Default value     | Mandatory   |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| MAXALERTAGE     | filter alerts to check those newer than this parameter (s)                                                                                                     | 86400             |             |
+| FILTERSERVER    | Define which devices sh ould be monitored based on the MachineName. This option willbe treated as a regular expression. By default all machine will be checked |                   |             |
+| WARNINGWARNING  | thresholds                                                                                                                                                      |                   |             |
```

Comment:
Capital T for all instances of "threshold" (and it should be threshold in the singular, shouldn't it?)